src/components/ExpandedNode/ExpandedNode.tsx

```diff
@@ -88,7 +95,7 @@ const ExpandedNode: FC<ExpandedNodeProps> = ({
             className={`${indexStyles.button} ${styles.defaultTextButton}`}
             onClick={(e): void => {
               e.preventDefault();
-              if (!comments.includes(defaultText)) setComments(comments + defaultText);
+              if (!comments?.includes(defaultText)) setComments(comments + defaultText);
```

Comment:
Since comments is initialized to be `note?.notes ?? ''` it will always be nonnull
```suggestion
              if (!comments.includes(defaultText)) setComments(comments + defaultText);
```

Reply:
Did you remove the `?` and then add it back in? `comments` is still initialized as `note?.notes ?? ''` so I still don't think it is needed.

Reply:
No, I think I just hosed that on the rebase. Fixed.